layout/PrimaryNav/PrimaryNavLink/index.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+import React from 'react'
+import classnames from 'classnames'
+import Icon from '../../../atoms/Icon'
+
+export default React.createClass({
+  displayName: 'PrimaryNavLink',
+
+  propTypes: {
+    icon: React.PropTypes.string,
+    active: React.PropTypes.bool,
+    isDashboardLink: React.PropTypes.bool,
+    clickHandler: React.PropTypes.func
+  },
+
+  render: function() {
+    var className = classnames({
+      'active': this.props.active,
+      'Navigation__dashboardLink': this.props.isDashboardLink
+    }, 'Navigation__link');
+
+    return (
+      <a href="#" className={ className } onClick={ this.props.clickHandler }>
+        <Icon icon={ this.props.icon } />
+        <span className="Navigation__linkText">
+          { this.props.children }
+        </span>
+      </a>
+    );
+  }
+});
```

Comment:
Spoke to @slwen we are leaving it as a full/normal/whatever component to take advantage of the proptypes.